public/popup.html

```diff
@@ -399,6 +399,17 @@ <h3>Make YouTube your own!</h3>
           <span class="slider round"></span>
         </label>
       </div>
+      <div class="checkbox-container">
+        <label for="checkbox-container">Auto Expand Video Chapters</label>
```

Comment:
The `for` attribute should the same as the `id`, so here it should be `for="expand-video-chapters"`, otherwise when you click on the text it will not enable/disable.

Reply:
In my testing it actually doesn't matter, the buttons work anyway. I think this is because it is just a `<label>` tag and the thing that triggers the action is the `<input>`. Maybe I'm missing something but I think that the `for=` in the `<label>` are redundant. Is it @Pedro-Gregorio ?

Reply:
Consistency wise, it matters! :D
Also, the `for` attribute on a label is a representation of which `input` we're targetting. You can see more here: [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for#usage).

Reply:
> Consistency wise, it matters! :D Also, the `for` attribute on a label is a representation of which `input` we're targetting. You can see more here: [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for#usage).

I see. Thanks for clarifying!